README.md

```diff
@@ -55,8 +55,14 @@ contains documents describing how to build and contribute to Electron.
 
 ## Quick Start
 
-Clone and run the [`atom/electron-quick-start`](https://github.com/atom/electron-quick-start)
-repository to see a minimal Electron app in action.
+```bash
+# Clone the electron-quick-start repository
+$ git clone https://github.com/atom/electron-quick-start
```

Comment:
It’d make sense to have these as `git clone https://github.com/atom/electron-quick-start && cd electron-quick-start`. That way, there’d be a clear separation: one two-part command for setting up, and another to run.